### ISSUE TITLE
Remove condname from environment

### ISSUE THREAD

**Issue by hmgaudecker** [hmgaudecker/econ-project-templates#71]
```
EnvironmentSectionNotValid: The following section on 'xxx/stata_demo/environment.yml' is invalid and will be ignored:
 - condname
```
Not sure what that is supposed to do?

**Comment by raholler** [comment]
How/ when does this error occur?

**Comment by hmgaudecker** [comment]
Upon project creation. Just a warning, but it is confusing people at a very early stage...

**Comment by raholler** [comment]
which conda version? never saw that one

**Comment by hmgaudecker** [comment]
I am on 4.7.12, not sure about the student where I also saw it (conda shell on Windows). I never saw that thing, is it maybe specific to the powershell ?

**Comment by raholler** [comment]
just reproduced this error.

**Comment by raholler** [comment]
fixed in commit 813dc2efa8eab2ac1397a826cd3e8e7447f50b68
